Formula/umlet.rb

```diff
@@ -0,0 +1,34 @@
+class Umlet < Formula
+  desc "This UML tool aimed at providing a fast way of creating UML diagrams."
+  homepage "http://www.umlet.com/"
+  url "http://www.umlet.com/umlet_14_2/umlet-standalone-14.2.zip"
+  sha256 "dc09538d04cb899218f3bdfdb5205f60359cff68cb1428d1228c6d91743d8cb9"
+
+  bottle :unneeded
+
+  depends_on :java => "1.6+"
+
+  def install
+    rm Dir["*.{desktop,exe}"]
+
+    libexec.install Dir["*"]
+
+    # Replace 'java' command in shell script for use the 'JAVA_HOME' env.
```

Comment:
Can remove this comment; self-explanatory from the code. Then we're good to pull I think 👍